jobs/JGDAS_FIT2OBS

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "fit2obs" -c "base fit2obs"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gdas"}}
+
+export xdate=$(${NDATE} -${VBACKUP_FITS} ${CDATE})
+export CDATE=${xdate} # Reset CDATE to xdate for lookback
```

Comment:
Is the external package using `$CDATE` or `$xdate`? Or neither? Don't export things that aren't needed downstream.

Reply:
The Fit2Obs package uses `CDATE` throughout whereas `xdate` is just used at the workflow level to reset `CDATE` to the lookback cycle and to set `vday` and `vcyc` before invoking the Fit2Obs scripts. A lot of this is lifted from how we were doing it in the vrfy job, it can be rethought if needed.

Reply:
I would just remove replace xdate with CDATE then, although I don't like fit2obs using CDATE rather than VDATE.

Reply:
Good call, done at 3b48ae11. Thanks!